dev-scripts/make-release.sh

```diff
@@ -82,7 +82,7 @@ done
 mkdir "$INSTALLER_BIN_DIR"
 # Construct the main script
 # Copy the shebang line in
-head  "$IN_DEV_MAIN_SCRIPT" >> "$INSTALLER_MAIN_SCRIPT"
+head -n 1 "$IN_DEV_MAIN_SCRIPT" >> "$INSTALLER_MAIN_SCRIPT"
```

Comment:
A bug in the install script ended up duplicating lines in the resulting install script.

Reply:
I'd say we should leave this for another pull request. The import version hasn't been updated in the main branch and a couple small fixes like that and this could easily make a good PR

Reply:
I'd rather pull in the import version separately since that is an update specifically for RITA 2.
Any other RITA 2 changes we need to make could be rolled into it though.
I'll put up a new PR with this fix.